.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: true
+dist: trusty
+language: python
+python:
+  - '3.5'
+before_install:
+  - "sudo apt-get update && sudo apt-get install -y --no-install-recommends texlive-fonts-recommended texlive-latex-extra texlive-fonts-extra dvipng texlive-latex-recommended latexmk poppler-utils latex-xcolor lmodern texlive-xetex texlive-generic-recommended texlive-full"
+  - "pip install 'lander==0.1.0b7'"
+script:
+  - "export TEXMFHOME=./lsst-texmf/texmf"
+  - "make"
+after_success:
+  - echo $ABSTRACT
+  - "lander --pdf dm_stack_releases*.pdf --upload --lsstdoc dm_stack_releases.tex --env=travis --ltd-product $PRODUCT --handle $HANDLE --abstract \"${ABSTRACT}\""
+env:
+  global:
+    - ABSTRACT="This attempts to summarise the debate around, and suggest a path forward, for LSST software releases. Although some recommendations are made in §6, they are intended to serve as the basis of discussion, rather than as a complete solution. This material is based on discussions with several team members over a considerable period. Errors are to be expected; apologies are extended; corrections are welcome."
+    - HANDLE="DMTN-044"
+    - PRODUCT="dmtn-044"
```

Comment:
@timj product is a LSST the Docs concept among other things defines the sub-domain name. We make our domains lower case to follow convention.